tests/unit/test_mcp_gateway_cli.py

```
@@ -96,6 +97,16 @@ def test_invalid_json_emits_fallback_ask_and_exit_2() -> None:
     assert body["decision"] == "ask"
 
 
+def test_argparse_error_emits_fallback_ask_and_exit_2() -> None:
+    # --json-io is required; omitting it triggers argparse error
+    code, out, err = _run_cli_with_input('{"tool_name": "bash"}', argv=[])
```

Ruff (0.15.13)

[warning] Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)